requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+Sphinx==1.4
+sphinx-autobuild==0.6.0
+sphinx-rtd-theme==0.1.9
+sphinxcontrib-asyncio==0.1.1
```

Comment:
Worth to upgrade to 0.2.0 (just released) as there autodoc support was added and you use it heavily.